phpstan.neon

```diff
@@ -0,0 +1,15 @@
+includes:
+    - phpstan-baseline.neon
+
+parameters:
+    paths:
+        - .
+
+    excludePaths:
+        - vendor
+        - Tests
```

Comment:
Don't you wanna analyze your tests?

Reply:
sure, but phpstan didn't like prophecy...
re-added tests and included prophecy plugin for phpstan...